index.html

```diff
@@ -1810,6 +1820,24 @@ <h2>Definition of Roles</h2>
 				    &lt;/div&gt;
 				    &lt;!-- ... --&gt;
 				</pre>
+				
+				<p>If the <code>caption</code> represents a long-form description, or if the description contains semantic elements which are important in understanding the description, authors MAY instead specify <pref>aria-labelledby</pref> to reference an element within the <code>caption</code> that represents the "name" of the parent element, and specify <pref>aria-details</pref> to reference an element within the <code>caption</code> that contains the descriptive content.</p>
```

Comment:
For this (and sort-of for the previous example), why would `aria-labelledby` need to reference an element within the caption?

For example, take this table that is named by a heading preceding it, and described or details-ed by the caption:

```html
<h2 id="table-name">Prices</h2>
<div role="table" aria-labelledby="table-name" aria-describedby/details="table-caption">
  <div role="caption">All prices in this table are blah blah blah something boring and long</div>
  <!-- table rows -->
</div>
```

I wonder if it'd be good to modify one of the details/desc examples to have the caption _only_ provide a description with the name coming from elsewhere, and update the wording to not specify that `aria-labelledby` needs to point within the caption

Reply:
Another example that illustrates this makes sense to me... so long as we aren't getting too far into APG example territory?

the intent of the caption being to provide a name, description or both is why the wording is written the way it is.  But to your point, someone could go and use a caption purely for descriptive purposes and get a name from somewhere else or use an aria-label... or leave it nameless if ... REASONS ....  so yeh.  one more example probably makes sense and then future updates could go into APG / some of this could go to APG if/when needed.